content/en/docs/setup/upgrade/index.md

```diff
@@ -90,6 +92,30 @@ $ istioctl proxy-config endpoints ${pod_name}.test-ns --cluster xds-grpc -ojson
 
 The output confirms that the pod is using `istiod-canary` revision of the control plane.
 
+### Uninstall old control plane
+
+After upgrading both control plane and data plane, you can uninstall old control plane. For example the following command uninstalls control plane with `revision=default`
```

Comment:
Grammer.  "After upgrading both the control plane and data plane, you can uninstall the old control plane...."

Reply:
done